src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -568,7 +568,7 @@ When Sentry sees this, we’ll automatically annotate the matching issue with a
 
 ### GitLab
 
-Sentry’s GitLab integration helps you find and fix bugs faster by using data from your GitLab commits. Additionally, you can streamline your triaging process by creating a GitLab issue directly from Sentry.
+Sentry’s GitLab integration helps you find and fix bugs faster by using data from your GitLab commits. Additionally, you can streamline your triaging process by creating a GitLab issue directly from Sentry. [Troubleshooting](https://docs.sentry.io/workflow/integrations/global-integrations/#troubleshooting-2)
```

Comment:
@thinkocapo Is Gitlab on prem? Did not know that...

Reply:
I heard they can do on-prem or saas. I see it now on their pricing page. Is our documentation for the SaaS version? I thought it was, but I see a  installation URL screen.

https://docs.sentry.io/workflow/integrations/global-integrations/#gitlab

If so, I could write "Troubleshooting for On-Premise" instead of just "Troubleshooting"

thoughts @jonesphillip ?

Reply:
I honestly don't know...

Reply:
Ya Gitlab can be on-prem